RNG.py

```diff
@@ -0,0 +1,33 @@
+import random
+
+print ("Welcome to the random name generator!")
+
+def get_shortlist():
+# Reads the names that th user inputs and returns a list.
+    shortlist_str = input("Please enter names, separated by commas: ")
+    return [name.strip() for name in shortlist_str.split(',')]
+    
+def get_num_names():
+# Reads the number of names to be selected from the list and returns the interger.
```

Comment:
Could you add more comprehensive docstrings?

Reply:
i.e. function descriptions, parameters (in this case none!) and what is returned